### PR TITLE
Remove build pipelines.

### PR DESCRIPTION
**Pull request by PFedak**

### CODE REVIEW

File: src/internal/metrics/metrics.proto [hunk 1]
```diff
@@ -22,41 +22,40 @@ message Metrics {
...
-    float resource_cpu_limit       = 45; // Total CPU limits set across all pipelines
-    float resource_cpu_limit_max   = 46; // Max CPU limit set
-    string resource_mem_limit      = 47; // String of memory limits set
-    int64 resource_gpu_limit       = 48; // Number of pipelines with
-    int64 resource_gpu_limit_max   = 49; // Max GPU limit set
-    string resource_disk_limit     = 50; // String of disk limits set across all pipelines
-    uint64 max_parallelism         = 51; // Max parallelism set
-    uint64 min_parallelism         = 52; // Min parallelism set
-    uint64 num_parallelism         = 53; // Number of pipelines with parallelism set
+    int64 cfg_egress               = 17; // Number of pipelines with Egress configured
```
Comment (nitinjainsj) [inline src/internal/metrics/metrics.proto:25]: I would not change the number. Just mark the field unused. An old interpreter will read "17" as data for pps_build.

**Review by nitinjainsj — COMMENTED** [review]
In a separate PR, we should remove build pipeline from top level make file, the corresponding directories in etc/pipeline-build, and goreleaser/docker-build-pipeline.yml

File: src/internal/metrics/metrics.proto (hunk 1 above)
Comment (PFedak) [reply src/internal/metrics/metrics.proto:25]: Hmm, I thought the plan was to make protos explicitly incompatible between 1.x and 2.x, but I can preserve the numbers

**Comment by chainlink** [comment]
Follow on PR: https://github.com/pachyderm/pachyderm/pull/6289

**Review by nitinjainsj — APPROVED** [review]

**Pull request merged**